engine/client.go

```diff
@@ -53,6 +52,7 @@ type (
 		arcConfig                  chainmodels.ARCConfig  // Configuration for ARC
 		bhsConfig                  chainmodels.BHSConfig  // Configuration for BHS
 		feeUnit                    *bsv.FeeUnit           // Fee unit for transactions
+		migrationsDisabled         bool                   // Disable automatic migrations
```

Comment:
Do we need this? we don't set it to true anywhere (neither through config, nor in tests)

Reply:
Apparently, after my removal of sqlmock-based tests - this turned out to be unused. 
Removing as you suggest 👍